diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+wdi/try
```

Comment:
remember, `pwd` will print the entire working directory so `/Users/grace/wdi/try`